javascript/index.js

```diff
@@ -9,20 +9,23 @@ let minDec = document.getElementById('minDec');
 let minUni = document.getElementById('minUni');
 let secDec = document.getElementById('secDec');
 let secUni = document.getElementById('secUni');
-let milDec = document.getElementById('milDec');
-let milUni = document.getElementById('milUni');
+// let milDec = document.getElementById('milDec');
+// let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+   printMinutes()
+  // function printSeconds()
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerHTML = Number(minutes[0])
```

Comment:
minutes doesn't exist in this code, the idea here was to use the methods available inside the chronometer element (getSeconds, getMinutes and twoDigitsNumber to get the needed results to update the DOM.